comment/templates/comment/comments/content.html

```diff
@@ -1,9 +1,12 @@
-<p class="js-comment-content {% block c_text_cls %}mb-0{% endblock c_text_cls %}">
+<p class="js-comment-content {% block c_text_cls %}mb-0{% endblock c_text_cls %}" id="{{ urlhash }}">
     {% if text_2 %}
         {{ text_1 }}
         <span class="d-none">{{ text_2 }}</span>
         <button class="js-read-more-btn btn btn-link btn-xs" style="font-size: 14px">read more...</button>
     {% else %}
         {{ text_1 }}
     {% endif %}
+    {% block comment_urlhash %}
+    {% include "comment/comments/urlhash.html" with urlhash=urlhash %}
+    {% endblock comment_urlhash %}
```

Comment:
I would prefer to add the permalink icon to the beginning of the comment with slightly reducing its size.
```suggestion
    {% block comment_urlhash %}
    {% include "comment/comments/urlhash.html" with urlhash=urlhash %}
    {% endblock comment_urlhash %}
    {% if text_2 %}
        {{ text_1 }}
        <span class="d-none">{{ text_2 }}</span>
        <button class="js-read-more-btn btn btn-link btn-xs" style="font-size: 14px">read more...</button>
    {% else %}
        {{ text_1 }}
    {% endif %}
```

Reply:
That would make the icon appear to the left. It would be different from what I've seen at different places.

Reply:
Yes I know. Let's be special case 😄

Reply:
I tried it and looks cool

<img width="903" alt="Screenshot 2020-07-02 at 13 45 35" src="https://user-images.githubusercontent.com/25413382/86355037-53ff0a80-bc6a-11ea-85be-7afb2008d2c2.png">

Reply:
It just looks different and a bit confusing to me. I'm short of better words right now :)

Also, you probably missed this part in my comment above
> Another issue with providing the permalink icon only on hover is that it will restrict its access for touch devices.

Reply:
We should not implement the hovering feature

Reply:
What size and padding have you used for the icon?

Reply:
I used your icon without changing any styles.